core/tests/protractor_desktop/extensions.js

```diff
@@ -88,12 +107,15 @@ describe('rich-text components', function() {
           title: 'title 1',
           content: await forms.toRichText('contents 2')
         }]);
+        await richTextChecker.readRteComponent('Svgdiagram', 'rectangle');
       });
 
     await explorationEditorPage.discardChanges();
     await users.logout();
   });
 
+
+
```

Comment:
Can you remove these? I don't think we need 3 blank lines of separation

Reply:
Done.